spk/octoprint/src/installer.sh

```diff
@@ -33,6 +33,9 @@ postinst ()
     # Install the wheels/requirements
     ${INSTALL_DIR}/env/bin/pip install --use-wheel --no-deps --no-index -U --force-reinstall -f ${INSTALL_DIR}/share/wheelhouse -r ${INSTALL_DIR}/share/wheelhouse/requirements.txt > /dev/null 2>&1
 
+    # Install OctoPrint
+    cd ${INSTALL_DIR}/share/${DNAME} && ${INSTALL_DIR}/env/bin/python setup.py install > /dev/null 2>&1
```

Comment:
This assumes DNAME of the package is the same name as the git repo. I think I'd prefer hardcoded OctoPrint here instead. This is more readable that using a variable that's not meant for that use.
DNAME refers to Display Name, it's the package name as displayed in the package center and for any GUI rendering.